server/src/test/java/com/cloud/server/StatsCollectorTest.java

```diff
@@ -511,4 +511,16 @@ public void testPersistVolumeStatsKVM() {
     public void testPersistVolumeStatsVmware() {
         performPersistVolumeStatsTest(Hypervisor.HypervisorType.VMware);
     }
+
+    private Map<String, String> convertJsonToMap(String json) {
+        Map<String, String> jsonMap = new HashMap<String, String>();
+        String[] keyValuePairs = json.replaceAll("[{}\"]", "").split(",");
```

Comment:
Do we need to replaceAll? It will clean up the string before adding to the hashmap, but if the same formatting is present in both expected and actual strings, then does it matter?

Reply:
To convert the json string to hashmap I want to remove the '{' and '}' characters and also all of the " characters(which is why replace all), so that we have a resulting comma separated string. It can be done using just `json.replace("{","").replace("}","").split(",")`, but with regex it seemed better. Each element of the comma separated string has a key value pair. Only by removing these characters we can get to the map representation. I had done all this just to avoid importing json assert and felt that this kind of implementation should be sufficient given that only these tests will be using it

Reply:
Have addressed in revision

Reply:
Thank you. Fix looks good to me.